src/neynar-api/v2/client.ts

```diff
@@ -41,7 +40,7 @@ import {
   BulkCastsResponse,
   FnameApi,
   FnameAvailabilityResponse,
-  FramesApi,
+  FrameApi,
```

Comment:
Please create a new section for `-- Frame --` in both the files and move all the methods associated to frame in there